apps/web/src/components/forms/password.tsx

```diff
@@ -89,12 +101,28 @@ export const PasswordForm = ({ className }: PasswordFormProps) => {
           Password
         </Label>
 
-        <Input
-          id="password"
-          type="password"
-          className="bg-background mt-2"
-          {...register('password')}
-        />
+        <div className="relative">
+          <Input
+            id="password"
+            type={showPassword ? 'text' : 'password'}
+            className="bg-background mt-2"
```

Comment:
```suggestion
            className="bg-background mt-2 pr-10"
```

Reply:
Yeah, that makes sense as well. Sent a fix for this :)